packages/nuxt/README.md

```diff
@@ -0,0 +1,20 @@
+# Onyx Nuxt
```

Comment:
please place the description in the docs app, so it appears here in the docs
![image](https://github.com/SchwarzIT/onyx/assets/151019977/094eb337-3ebc-40c3-a6a8-09587421c946)
like http://localhost:5173/development/packages/figma-utils.html

copy the content of this file into 
![image](https://github.com/SchwarzIT/onyx/assets/151019977/2bdce71d-ebb8-465a-bdfc-6b188d7a26ba)

the readme of the actual packages are pointing to the docs with a logo on top, compare to
 `packages/figma-utils/README.md`